### DISCUSSION TITLE
快速收藏无法关闭&启动应用报错

### DISCUSSION THREAD

**Discussion by lucifere02** [ywmoyue/biliuwp-lite#811] [bug]
### Bug 描述

我不需要收藏到默认收藏夹，所以在设置里关闭了快速收藏功能，但是依然每次收藏都会自动收藏到默认收藏夹，重启软件也一样。
另外还有一个疑似bug，但是对使用似乎没有影响，就是每次打开软件首页都会有一行字：程序出现一个错误，已记录。或者：出现一个未处理错误，已记录。已附带日志文件。
注：截图和日志均为该疑似bug相关，和快速收藏功能无关。

### 操作步骤

```plain text
如上所述。
```

### 期望结果

应当能够关闭快速收藏功能。
以及一个优化建议，能否提供自定义快速收藏对应的收藏夹功能，而不是只能收藏到默认收藏夹。

### 截图

![image](https://github.com/user-attachments/assets/c9e82a58-4473-4190-8302-e47ed2bd92a1)
![image](https://github.com/user-attachments/assets/a57872f5-5eaf-4853-8c82-fa4df17725b1)

### 日志

2024-08-30 23:02:49.4535|ERROR|3|App.App_UnhandledException|程序运行出现错误|系统找不到指定的文件。 (Exception from HRESULT: 0x80070002)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

Windows 11 21H2 (Build 22000)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
需要将日志级别设置为Trace后看这行日志附近的日志上下文，只看这一行日志什么都看不出来

**Comment by ywmoyue** [comment]
issue包含多个问题，转discussion

另外开新的issue https://github.com/ywmoyue/biliuwp-lite/issues/812
